src/Api/OperationTypeDeprecationHelper.php

```diff
@@ -38,7 +38,7 @@ public static function getOperationType($operationType): string
         if (is_bool($operationType)) {
             @trigger_error('Using a boolean for the Operation Type is deprecrated since API Platform 2.1 and will not be possible anymore in API Platform 3', E_USER_DEPRECATED);
 
-            $operationType = $operationType === true ? OperationType::COLLECTION : OperationType::ITEM;
+            $operationType = true === $operationType ? OperationType::COLLECTION : OperationType::ITEM;
```

Comment:
`true ===` is useless here!

Reply:
though this is my bad haha, how did this end up in the code :P.

Reply:
It looks like PHP-CS-Fixer has a new rule for Yoda style 😛

Reply:
We should probably apply it on lower branches.

Reply:
Yes it was PHP-CS-Fixer